yarn.lock

```diff
@@ -15203,7 +15203,7 @@ preact@10.15.1:
     core-js "^3.13.0"
     core-js-pure "^3.13.0"
     criteo-direct-rsa-validate "^1.1.0"
-    crypto-js "^3.3.0"
+    crypto-js "^4.2.0"
```

Comment:
Is this correct?

Reply:
This is correct it came from a rebase.

Reply:
We should probably not commit this change as part of this PR. It looks like this weird flop-flopping issue we occasionally see when Yarn gets confused with the github path for our fork of prebid.js.

Reply:
Think this is the right version and has since been fixed on main by: https://github.com/guardian/dotcom-rendering/pull/9475

So another rebase should remove the diff from this PR.

Reply:
Rebase has sorted this 👍